test/test_shallow_water_models.jl

```diff
@@ -97,5 +119,17 @@ end
                 @test time_stepping_shallow_water_model_works(arch, topos[1], nothing, advection)
             end
         end
+
+        for timestepper in (:RungeKutta3, :QuasiAdamsBashforth2)
+            @testset "Time-stepping ShallowWaterModels [$arch, $timestepper]" begin
+                @info "  Testing time-stepping ShallowWaterModels [$arch, $timestepper]..."
+                @test time_stepping_shallow_water_model_works(arch, topos[1], nothing, nothing, timestepper=timestepper)
+            end
+        end
+
+        @testset "ShallowWaterModel with tracers and forcings [$arch]" begin
+            @info "  Testing ShallowWaterModel with tracers and forcings [$arch]..."
+            shallow_water_model_tracers_and_forcings_work(arch)
+        end
```

Comment:
Ah, somehow I missed these in my review. Got it.

Reply:
Thanks for looking them over